install/install.sh

```diff
@@ -47,8 +47,8 @@ usage() {
          -e install_server=false
     * Skipping install for CLI:
          -e install_cli=false
-    * Specify server port (defaults to 443):
-         -e server_port=8443
+    * Specify server port (defaults to 9443):
+         -e server_port=9443
```

Comment:
This option is not as big of a deal as the above, but if the user is supplying the ```-e server_port=*``` option, they probably want to override the default, so 8443 or a different number could be used here as well.